blurb/blurb.py

```diff
@@ -495,10 +495,10 @@ def _parse_next_filename(filename):
         components = filename.split("/")
         section, filename = components[-2:]
         section = unsanitize_section(section)
-        assert section in sections, f"Unknown section {section}"
+        assert section in sections, "Unknown section {}".format(section)
 
         fields = [x.strip() for x in filename.split(".")]
-        assert len(fields) >= 4, f"Can't parse 'next' filename! filename {filename!r} fields {fields}"
+        assert len(fields) >= 4, "Can't parse 'next' filename! filename {!r} fields {}".format(filename, fields)
```

Comment:
The PEP 8 allows up to 120 characters. The file already contains lines longer than 120 characters (before my change) ;-)